inst/tests/tests.Rraw

```diff
@@ -18532,3 +18532,15 @@ test(2260.15, inherits(tryCatch(messagef("x", class="test_message"), condition=i
 test(2260.16, inherits(tryCatch(messagef("x", class="test_message"), condition=identity), "message"))
 test(2260.17, inherits(tryCatch(packageStartupMessagef("x", class="test_psm"), condition=identity), "test_psm"))
 test(2260.18, inherits(tryCatch(packageStartupMessagef("x", class="test_psm"), condition=identity), "packageStartupMessage"))
+
+test(2261.01, setNumericRounding(0L), 0L)
```

Comment:
better to initialize this -- it could be that some other test earlier in the suite is changed with adverse effects here. e.g.

```r
old_rounding = getNumericRounding()
setNumericRounding(0L)
test(...)
...
setNumericRounding(old_rounding)
```